playground/milvus/MilvusPlayground.ApiService/Program.cs

```diff
@@ -0,0 +1,131 @@
+using Milvus.Client;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add service defaults & Aspire components.
+builder.AddServiceDefaults();
+
+builder.AddMilvusClient("milvus");
+
+// Add services to the container.
+builder.Services.AddProblemDetails();
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+app.UseExceptionHandler();
+
+app.MapDefaultEndpoints();
+
+app.MapGet("/create", async (MilvusClient milvusClient, ILogger<Program> logger) =>
+{
+    string collectionName = "book";
+    MilvusCollection collection = milvusClient.GetCollection(collectionName);
+
+    //Check if this collection exists
+    var hasCollection = await milvusClient.HasCollectionAsync(collectionName);
+
+    if (hasCollection)
+    {
+        await collection.DropAsync();
+        Console.WriteLine("Drop collection {0}", collectionName);
+    }
+
+    await milvusClient.CreateCollectionAsync(
+                collectionName,
+                new[] {
+                FieldSchema.Create<long>("book_id", isPrimaryKey:true),
+                FieldSchema.Create<long>("word_count"),
+                FieldSchema.CreateVarchar("book_name", 256),
+                FieldSchema.CreateFloatVector("book_intro", 2)
+                }
+            );
+    logger.LogInformation("Collection created: book");
+    return Results.Ok("Collection created");
+});
+
+app.MapGet("/insert", async (MilvusClient milvusClient, ILogger<Program> logger) =>
```

Comment:
Would it make sense to combine this with `/create`? That way I only need to hit 1 endpoint to set the thing up, and then `/search` to ensure it is working. Like how you did in

https://github.com/dotnet/aspire/blob/756dfe663d0759aa0d2a0e1208502a403a398c32/playground/Qdrant/Qdrant.ApiService/Program.cs#L19